packages/azure-http-specs/specs/azure/client-generator-core/usage/main.tsp

```diff
@@ -90,11 +91,31 @@ namespace ModelInOperation {
   op modelInReadOnlyProperty(@body body: RoundTripModel): {
     @body body: RoundTripModel;
   };
+
+  @doc("""
+    Serialize the 'OrphanModel' as request body.
+    
+    Expected body parameter: 
+    ```json
+    {
+      "name": "name",
+      "desc": "desc"
+    }
+    ```
+    """)
+  @global.Azure.ClientGenerator.Core.convenientAPI(false)
+  @route("/orphanModelSerializable")
+  @put
+  op orphanModelSerializable(@body body: unknown): NoContentResponse;
 }
 
 @doc("Not used anywhere, but access is override to public so still need to be generated and exported with serialization.")
 @global.Azure.ClientGenerator.Core.usage(global.Azure.ClientGenerator.Core.Usage.input)
 @global.Azure.ClientGenerator.Core.access(global.Azure.ClientGenerator.Core.Access.public)
 model OrphanModel {
+  @global.Azure.ClientGenerator.Core.clientName("modelName")
   name: string;
+
+  @encodedName("application/json", "desc")
+  description: string;
 }
```

Comment:
I've updated the model a little, to mix typespec name, client name, encoded name. And we expect SDK find the correct name for serialization.